generators/generator-bot-adaptive/generators/app/templates/dotnet/functions/Triggers/StaticFilesTrigger.cs

```diff
@@ -0,0 +1,66 @@
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.StaticFiles;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+
+namespace <%= botName %>.Triggers
+{
+    public class StaticFilesTrigger
+    {
+        private const string StaticFilesDirectory = "www";
+
+        private readonly FileExtensionContentTypeProvider _provider;
+
+        public StaticFilesTrigger()
+        {
+            var provider = new FileExtensionContentTypeProvider();
+            provider.Mappings[".lu"] = "application/vnd.microsoft.lu";
+            provider.Mappings[".qna"] = "application/vnd.microsoft.qna";
```

Comment:
What about manifest files extensions? Do we need to provide that as well?

Reply:
FileExtensionContentTypeProvider contains a default mapping for many different file extensions to corresponding content types. As the schema files are just .json, these are already covered by the default mapping. The way this code is written just supplements that by adding the additional ones needed for our scenarios.